app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"],
+                          person["age"]) for person in people]
+    for i, person in enumerate(people):
```

Comment:
"i" is not descriptive. And also, you don't really need `enumerate` or `range` in your loop

Reply:
fixed